doc/running-docker.md

```diff
@@ -1,22 +1,21 @@
+# Running the PrairieLearn docker image
 
-# Installing and running PrairieLearn
-
-* Step 1: Install [Docker Community Edition](https://www.docker.com/community-edition). It's free. 
+1. Install [Docker Community Edition](https://www.docker.com/community-edition). It's free.
```

Comment:
Remove this link -- I've noticed it's confusing for people because we have better links on the sub-bullet points below.